view/adminhtml/ui_component/allegro_offer_form_create.xml

```diff
@@ -74,9 +74,11 @@
                     <item name="visible" xsi:type="boolean">true</item>
                     <item name="dataType" xsi:type="string">text</item>
                     <item name="formElement" xsi:type="string">input</item>
+                    <item name="component" xsi:type="string">Macopedia_Allegro/js/allegro_offer/form/field/name-length-counter</item>
+                    <item name="elementTmpl" xsi:type="string">Macopedia_Allegro/allegro_offer/form/field/name-length-counter</item>
                     <item name="validation" xsi:type="array">
                         <item name="required-entry" xsi:type="boolean">true</item>
-                        <item name="max_text_length" xsi:type="number">50</item>
+                        <item name="allegro-name-length" xsi:type="boolean">true</item>
```

Comment:
item "lenght" and type "boolean" doesn't sound valid?

Reply:
I can change name of this validator to "allegro-name-validator". Is it fine?

Reply:
ok, seems that this is how magento name things ;)